### PR TITLE
Add extra translations for popi user data line

### PR DESCRIPTION
**Pull request by KaitCrawford**

### CODE REVIEW

File: config/go-app-ussd_popi_user_data.afr_ZA.json [hunk 1]
```diff
@@ -209,19 +209,19 @@
     ],
     "Invalid phone number. Please re-enter (with no spaces)": [
         null,
-        ""
+        "Ongeldige selnommer. Tik asb weer in"
     ],
     "Sorry, the number you are trying to move to already has an active registration. To manage that registration, please redial from that number.": [
         null,
-        ""
+        "Jammer, die nommer wat jy nou wil gebruik het reeds 'n aktiewe registrasie. Om daardie registrasie te bestuur moet jy weer daardie nommer bel"
```
Comment (erikh360) [inline config/go-app-ussd_popi_user_data.afr_ZA.json]: This doesn't quite say the same thing as the english, the english says you should redial from that number and the afrikaans says you should redial that number.

File: config/go-app-ussd_popi_user_data.afr_ZA.json [hunk 2]
```diff
@@ -209,19 +209,19 @@
...
     ],
     "Thank you. Your info has been updated.": [
         null,
         "Baie dankie. Jou info is bygewerk."
     ],
     "Your personal information stored on MomConnect has not been removed.": [
         null,
-        ""
+        "Jou persoonlike info wat op MomConnect gestoor is, is verwyder"
```
Comment (erikh360) [inline config/go-app-ussd_popi_user_data.afr_ZA.json]: This means the exact opposite of the english 🙂 
"Jou persoonlike info wat op MomConnect gestoor is, is nie verwyder" - maybe

File: config/go-app-ussd_popi_user_data.afr_ZA.po [hunk 3]
```diff
@@ -250,21 +253,23 @@ msgstr ""
...
-msgstr ""
+msgstr "Ongeldige selnommer. Tik asb weer in"
 
 #: go-app-ussd_popi_user_data.js:497
 msgid ""
 "Sorry, the number you are trying to move to already has an active "
 "registration. To manage that registration, please redial from that number."
 msgstr ""
+"Jammer, die nommer wat jy nou wil gebruik het reeds 'n aktiewe registrasie. "
+"Om daardie registrasie te bestuur moet jy weer daardie nommer bel"
```
Comment (erikh360) [inline config/go-app-ussd_popi_user_data.afr_ZA.po]: Same as above

File: config/go-app-ussd_popi_user_data.afr_ZA.po [hunk 4]
```diff
@@ -250,21 +253,23 @@ msgstr ""
...
 
 #: go-app-ussd_popi_user_data.js:534
 msgid "Thank you. Your info has been updated."
 msgstr "Baie dankie. Jou info is bygewerk."
 
 #: go-app-ussd_popi_user_data.js:541
 msgid "Your personal information stored on MomConnect has not been removed."
-msgstr ""
+msgstr "Jou persoonlike info wat op MomConnect gestoor is, is verwyder"
```
Comment (erikh360) [inline config/go-app-ussd_popi_user_data.afr_ZA.po]: Same as above

**Review by erikh360 — COMMENTED** [review]
Technically the changes all look fine, I have some comments on the Afrikaans translation though.

**Comment by KaitCrawford** [comment]
Yeah I'm a bit grumpy about the translations. I had to fix some of the others as well. Going to raise it that someone needs to check them. Thanks for catching these

**Review by erikh360 — APPROVED** [review]

**Pull request merged**